src/python_testing/TC_AVSM_2_1.py

```diff
@@ -161,95 +164,112 @@ async def test_TC_AVSM_2_1(self):
                 endpoint=endpoint, cluster=cluster, attribute=attr.MaxConcurrentEncoders
             )
             logger.info(f"Rx'd MaxConcurrentEncoders: {value}")
+            asserts.assert_is_not_none(value, "MaxConcurrentEncoders is None")
+            matter_asserts.assert_valid_uint8(value, "MaxConcurrentEncoders")
 
         self.step(3)
         if self.pics_guard(self.check_pics("AVSM.S.A0001")):
             value = await self.read_single_attribute_check_success(
                 endpoint=endpoint, cluster=cluster, attribute=attr.MaxEncodedPixelRate
             )
             logger.info(f"Rx'd MaxEncodedPixelRate: {value}")
+            asserts.assert_is_not_none(value, "MaxEncodedPixelRate is None")
+            matter_asserts.assert_valid_uint32(value, "MaxEncodedPixelRate")
 
         self.step(4)
         if self.pics_guard(self.check_pics("AVSM.S.A0002")):
             value = await self.read_single_attribute_check_success(
                 endpoint=endpoint, cluster=cluster, attribute=attr.VideoSensorParams
             )
             logger.info(f"Rx'd VideoSensorParams: {value}")
+            asserts.assert_is_not_none(value, "VideoSensorParams is None")
```

Comment:
This also needs to test that the fields within the struct are valid.
SensorWidth is at least 64
SensorHeight is at least 64
MaxFPS is at least 1
MaxHDRFPS is between 1 and MaxFPS